lib/web_ui/dev/test_runner.dart

```diff
@@ -15,11 +15,27 @@ import 'package:test_api/src/backend/runtime.dart'; // ignore: implementation_im
 import 'package:test_core/src/executable.dart'
     as test; // ignore: implementation_imports
 
+import 'integration_tests_manager.dart';
 import 'supported_browsers.dart';
 import 'test_platform.dart';
 import 'environment.dart';
 import 'utils.dart';
 
+/// The type of tests requested by the tool user.
+enum TestTypesRequested {
+  /// For running the unit tests only.
+  unit,
+
+  /// For running the integration tests only.
+  integration,
+
+  /// For running both unit and integration tests.
+  all,
+
+  /// Run no tests.
+  none,
```

Comment:
What's the value of this mode? Can we remove it and instead make sure it's impossible to not run any tests at all?

Reply:
good question. done.